### PR TITLE
Matt/GitHub secrets

### PR DESCRIPTION
**Pull request by MattKersey**
You will need to add a .env file to frontend folder with a `STRIPE_API_KEY` field

### CODE REVIEW

**Comment by ceguiluzrosas** [comment]
LGTM 👍🏽

File: frontend/src/Shop/Shop.js [hunk 1]
```diff
@@ -93,7 +93,7 @@ class Shop extends React.Component {
   }
 
   handleCheckout (event) {
-    const stripe = Stripe('pk_test_51Hu2LSG8eUBzuEBEesc0XBxkWzW7RyHcK7ckx9DqgHL710Mh3BF0NRyowXw82xTUa8vBVIdamkyzkllOzHepdfPc00MoyUNiR6')
+    const stripe = Stripe(process.env.STRIPE_API_KEY)
```
Comment (ceguiluzrosas) [inline frontend/src/Shop/Shop.js:96]: How does this work? Do we create a .env file in `frontend/src`?

**Review by ceguiluzrosas — APPROVED** [review]
Just one question (above) @MattKersey

File: frontend/src/Shop/Shop.js (hunk 1 above)
Comment (MattKersey) [reply frontend/src/Shop/Shop.js:96]: It should be in `frontend`, same directory as package.json. All it needs is the `STRIPE_API_KEY` field.

**Pull request merged**